xml_inscriptions/transcribed/MI.AA.UM.KM.G.21066.xml

```diff
@@ -0,0 +1,132 @@
+                        <p>relief depicting a standing female figure and seated jackals on either side, facing the figure.</p>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               <history><!--summary, origin, provenance go here-->
+                  <summary>
+                     <rs/>  <!-- content can be "modern" or blank -->
+                  </summary>
+                  
+                  <origin>
+                     <date evidence="#archaeology" notBefore="0275" notAfter="0340">late third to
+                        early fourth century CE</date>
+                     <placeName ref="africa.egypt">Egypt, Terenouthis (Kom Abou Billou)</placeName>
+                  </origin>
+                  <provenance type="found" subtype="discovered">
+                     <p>Uncovered during excavations in modern Kom Abou Billou (the modern name of the necropolis of the ancient city of Terenouthis) conducted in February-April 1935 by the University of Michigan.</p>
+                  </provenance>
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#SEG_20"/>
+                  <biblScope unit="item">no. 561</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#KomAbouBillou_Hooper_1961"/>
+                  <biblScope unit="item">no. 32</biblScope>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include  href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2022-08-13" who="Sophia Papandonatou">added metadata, transcription, translation, commentary</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--facsimile element goes here--><text>
+      <body>
+         <div type="edition">
+         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+            <ab>
+               <lb n="1"/><persName><name key="Θουῶρις Thouoris">Θου<unclear>ῶ</unclear>ρις</name></persName> ἄωρε μην
+               <lb n="2" break="no"/><supplied reason="lost">ῶν</supplied> <gap reason="lost" quantity="1" unit="character"/>, ἐτῶν <num value="9" xml:lang="grc"><unclear>θ</unclear>ʹ</num>; <g type="ἔτους"/> <gap reason="omitted" quantity="1" unit="character" cert="low"/>, Τεῦβι<note>!</note> <num value="4" xml:lang="grc"><hi rend="supraline">δ</hi>ʹ</num>.
```

Comment:
Careful when using ; here -- I think you mean to use a raised dot/ano teleia, since ; is the Greek question mark.